README.adoc

```diff
@@ -327,7 +328,7 @@ spec:
 
 === Running the UI outside the cluster
 
-When developing the http://github.com/kiali/kiali/frontend[Kiali UI] you will find it useful to run it outside of the cluster to make it easier to update the UI code and see the changes without having to re-deploy. The preferred approach for this is to use the _proxy_ feature of React. The process is described https://github.com/kiali/kiali/frontend#developing[here].
+When developing the https://github.com/kiali/kiali-ui[Kiali UI] you will find it useful to run it outside of the cluster to make it easier to update the UI code and see the changes without having to re-deploy. The preferred approach for this is to use the _proxy_ feature of React. The process is described https://github.com/kiali/kiali-ui#developing[here].
```

Comment:
Revert this line change. Looks like this PR is not rebased on the latest master. We no longer use kiali-ui, this line was changed recently.

Reply:
I see, reverted!